locallib.php

```diff
@@ -1500,7 +1498,22 @@ protected function remove_links($htmltext) {
         }
         $config->set('Cache.SerializerPath', $cachedir);
         $config->set('Cache.SerializerPermissions', $CFG->directorypermissions);
-        $config->set('HTML.ForbiddenElements', array('script', 'style', 'applet', 'a'));
+        $config->set('HTML.ForbiddenElements', array('script', 'style', 'applet'));
```

Comment:
There is a close bug related to this feature PR
Issue https://github.com/bozoh/moodle-mod_simplecertificate/issues/111

I don't know if i approve this PR it's going to re-open this bug (auto-llink)

Can you test it ?, enabling auto link feature in you moodle